plugins/GSdx/GSHwHack.cpp

```diff
@@ -85,22 +85,18 @@ bool GSC_DBZBT3(const GSFrameInfo& fi, int& skip)
 		}
 		else if(fi.TME && (fi.FBP == 0x00000 || fi.FBP == 0x00e00 || fi.FBP == 0x01000) && fi.FPSM == PSM_PSMCT32 && fi.TPSM == PSM_PSMT8H)
 		{
-			if (Dx_only) { // Ought to be fine with blending accuracy (fbmask?)
+			if (Aggressive) { // Partially works. A small black line shakes at the bottom on dx9. Can be solved with x y offset or removing outlines with Aggressive crc.
 				if(fi.FBMSK == 0x00000)
 				{
-					skip = 28; // outline
-				}
-				if(fi.FBMSK == 0x00FFFFFF)
-				{
-					skip = 1;
+					skip = 28; // Character outlines
```

Comment:
I mentioned that in the above comment. The black line shaking at the bottom is the result of not full implementation yet. Aside from that the code can be purged but I left it there just in case people don't want to have the 50% outlines working and the shaking on dx9.
There are no other issues , maybe in the past it was connected with the vertical red lines but that issue has been fixed.